dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassToSymbolReferences.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableSetMultimap;
+import java.util.Objects;
+
+/**
+ * A set of symbol references from source class to target symbols (class, method, and field
```

Comment:
I'm not following this. This needs clarification in the javadoc and probably a different name.

Reply:
Renamed to `SymbolReferenceMaps` and enriched javadoc.